frontend/libs/studio-components/src/components/StudioAlert/StudioAlert.test.tsx

```diff
@@ -0,0 +1,16 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import type { StudioAlertProps } from './StudioAlert';
+import { StudioAlert } from './StudioAlert';
+
+describe('StudioAlert', () => {
+  it('should render the component', () => {
+    renderTestAlert({ severity: 'danger' });
+    const studioAlert = screen.getByText('Feil');
```

Comment:
Where does the text "Feil" comes from? 🤔 
```suggestion
    const studioAlert = screen.getByText('Feil') // <-- Feil;
```

Reply:
It's something coming from Designsystemet https://github.com/digdir/designsystemet/blob/616046938ab42234ff0f8a078d23455a8be7a0c8/packages/react/src/components/Alert/Alert.tsx#L26

Reply:
I found out this is controlled by the prop `iconText`. So i'm changing this test to use a specified value instead